scripts/update_config.py

```diff
@@ -0,0 +1,24 @@
+'''
+Overwrite a config file by renaming args.
+Require one argument: path_to_file.
+'''
+
+import sys
+
+# Mapping - key: old name, value: new name
+name_dict = {'task_patience':'lr_patience',\
+               'do_train': 'do_pretrain',\
```

Comment:
FYI, don't need the backslashes since each line ends with a comma. This should work:
```
name_dict = {'task_patience': 'lr_patience',
             'do_train': 'do_pretrain',
             ...
}
```